doc/man1/common/job-param-additional.rst

```diff
@@ -6,6 +6,15 @@
    option is ignored, though :man1:`flux-jobs` may display the queue
    name in its rendering of the ``{queue}`` attribute.
 
+.. option:: -B, --bank=NAME
+
+   Set the bank name for this job to ``NAME``. This option is equivalent
+   to `--setattr=bank=NAME`, and results in the ``bank`` attribute being
```

Comment:
Isn't it `--setattr=system.bank=NAME`?

Reply:
No. See [flux-submit(1)](https://flux-framework.readthedocs.io/projects/flux-core/en/stable/man1/flux-submit.html#cmdoption-flux-submit-setattr):

>  If KEY does not begin with system., user., or ., then system. is assumed.

Reply:
Ah OK.  All good then.